qiskit/qobj/_qobj.py

```diff
@@ -100,7 +101,7 @@ class Qobj(QobjItem):
         experiments (list[QobjExperiment]): list of experiments.
         header (QobjHeader): headers.
         type (str): experiment type (QASM/PULSE).
-        _version (str): Qobj version.
+        schema_version (str): Qobj version.
     """
 
     REQUIRED_ARGS = ['id', 'config', 'experiments', 'header']
```

Comment:
Yes - and it actually is (ie. it will always be present in the `Qobj` instance), but not configurable by the user (line 117). If seemed a reasonable assumption that `qiskit` will always produce `Qobj`s for the same (latest) version of the schema, at least in its current form.